xb.bat

```diff
@@ -29,12 +29,27 @@ REM Utilities
 REM ============================================================================
 
 :check_python
-SETLOCAL
+setlocal enabledelayedexpansion
 SET FOUND_PYTHON_EXE=""
-1>NUL 2>NUL CMD /c where python3
-IF NOT ERRORLEVEL 1 (
-  SET FOUND_PYTHON_EXE=python3
+FOR /F %%I IN ('where python') DO (
```

Comment:
Thanks for this fix - I tried it out and ran into issues because my paths have spaces in them, which caused FOR to truncate them. The following fix (based on https://stackoverflow.com/a/5553316) worked for me:
```suggestion
FOR /F "delims=" %%I IN ('where python') DO (
```
If you'd like to confirm this for yourself with an example, run
```
(ECHO example one & ECHO demo two) > test.txt 
FOR /F %A IN (test.txt) DO ECHO %A 
```
and then add `"delims="` after the `/F` and try it again.

Reply:
Looking at the continuous integration failure, I think it's caused by this same issue.